src/Data/Data.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Project\Data;
+
+class Data extends AbstractData
+{
+    /**
+     *@param string $type, string $color, int $capacity, string $fuel, int $price, string $comment
+     *@return array
+     */
+    public function get($type, $color, $capacity, $fuel, $price, $comment)
```

Comment:
if you are extending the Abstract data with public method dataProcessing($type, $color, $capacity, $fuel, $price, $comment)
your new method get($type, $color, $capacity, $fuel, $price, $comment) is doing exactly the same. Why do you need two absolutely equivalent public functions, that are returning the array of its input parameters? Could you implement some processing? But why the second function is called get?